Changelog.md

```diff
@@ -1,5 +1,7 @@
 Unreleased
 -------------------------
+[Enhancement] Allow to enable/disable displaying tests sdterr
```

Comment:
Typo, stderr.

Also, should be "test stderr output streams".

Reply:
Done.